homepage/scripts/utils.js

```diff
@@ -19,8 +19,15 @@ export const [setLibs, getLibs] = (() => {
     (prodLibs, location) => {
       libs = (() => {
         const { hostname, search } = location || window.location;
-        if (!['.hlx.', '.stage.', 'local'].some((i) => hostname.includes(i))) return prodLibs;
+        if (hostname.includes('stage.adobe.com')) {
+          return "https://stage--milo--adobecom.hlx.live/libs";
+        }
+        if (!['.hlx.', 'local'].some((i) => hostname.includes(i))) return prodLibs;
         const branch = new URLSearchParams(search).get('milolibs') || 'main';
```

Comment:
The current solution works 👍 
Here is a shorter alternative if you want to change:
```suggestion
       if (!['.hlx.', 'local'].some((i) => hostname.includes(i))) return prodLibs;
        const branch = new URLSearchParams(search).get('milolibs') || hostname.includes('.stage.') ? 'stage' : 'main';
```